utils.c

```diff
@@ -279,6 +279,27 @@ size_t janus_strlcat(char *dest, const char *src, size_t dest_size) {
 	return ret;
 }
 
+int janus_strlcat_fast(char *dest, const char *src, size_t dest_size, size_t *offset) {
+	if(dest == NULL || src == NULL || offset == NULL) {
+		JANUS_LOG(LOG_ERR, "janus_strlcat_fast: invalid arguments\n");
+		return -1;
+	}
+	if(*offset >= dest_size) {
+		JANUS_LOG(LOG_ERR, "janus_strlcat_fast: overflow\n");
+		return -2;
+	}
+	char *p = memccpy(dest + *offset, src, 0, dest_size - *offset);
+	if(p == NULL) {
+		JANUS_LOG(LOG_ERR, "janus_strlcat: truncation occurred, %lu >= %lu\n",
```

Comment:
nit: to avoid confusion, probably better to log `"janus_strlcat_fast:...`

Reply:
Ah that's a copy/paste typo, thanks for catching! Incidentally, since this is a `LOG_ERR`, I think the name of the function is actually redundant, since it will appear in the prefix anyway, e.g.:

    [ERR] [utils.c:janus_strlcat_fast:288] janus_strlcat_fast: overflow

so I'll just remove it from both methods.